ansible_base/resource_registry/views.py

```diff
@@ -141,6 +142,11 @@ def manifest(self, request, name, *args, **kwargs):
             return HttpResponseNotFound()
 
         resources = Resource.objects.filter(content_type__resource_type=resource_type).prefetch_related("content_object")
+
+        if name == "shared.user":
+            system_user = getattr(settings, "SYSTEM_USERNAME", "_system")
```

Comment:
If `not hasattr(settings, "SYSTEM_USERNAME")`, my preference is to not modify `resources`.